### PR TITLE
Elastic search configurable response type

### PR DESCRIPTION
**Pull request by dylman79**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

#986

### CODE REVIEW

**Comment by dylman79** [comment]
Separate PR for this (https://github.com/cube-js/cube.js/pull/992) but does not utilise elastic search api format setting. 
Happy to close if the original PR owner wants to amend theirs but feel it should use api as intended as well as cater for external setting via env or config

**Comment by igorlukanin** [comment]
Adding @hassankhan to review the changes to the docs.

File: docs/content/Configuration/Environment-Variables-Reference.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ databases [in this guide][link-connecting-to-db].
...
 | `CUBEJS_DB_URL`                                 | Druid, Elasticsearch | The URL for a database                                                                                                                                                                                      | A valid database URL for Druid/Elasticsearch                     |
 | `CUBEJS_DB_ELASTIC_OPENDISTRO`                  | Elasticsearch        | If `true`, then use the Open Distro for Elasticsearch                                                                                                                                                       | `true`, `false`                                                  |
+| `CUBEJS_DB_ELASTIC_QUERY_FORMAT`                | Elasticsearch        | By default, queries return data in JDBC format, but you can also return data in standard Elasticsearch jdbc, json, csv, yaml, or raw formats (only json or jdbc currently supported)                        | `json`, `jdbc`
```
Comment (hassankhan) [inline docs/content/Configuration/Environment-Variables-Reference.md:58]:
```suggestion
| `CUBEJS_DB_ELASTIC_QUERY_FORMAT`                | Elasticsearch        | By default, queries return data in JDBC format, but you can also return data in standard Elasticsearch JDBC, JSON, CSV, YAML or raw formats (only JSON and JDBC are currently supported)                    | `json`, `jdbc`                                                   |
```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Minor nitpick aside, looks great 👍

**Comment by dylman79** [comment]
Changes applied - Thanks

**Review by hassankhan — APPROVED** [review]

**Comment by hassankhan** [comment]
LGTM, just pinging @cube-js/core for a review to make sure we didn't miss anything 😀

**Review by ovr — APPROVED** [review]
LGTM

**Pull request merged**